HeliPort/Appearance/WiFiMenuItemView.swift

```diff
@@ -85,47 +82,48 @@ class WifiMenuItemView: NSView {
+            layoutSubtreeIfNeeded()
         }
     }
 
-    func setupLayout() {
-        menuItemView.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
-        menuItemView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
-
-        statusImage.centerYAnchor.constraint(equalTo: menuItemView.centerYAnchor).isActive = true
-        statusImage.leftAnchor.constraint(equalTo: menuItemView.leftAnchor, constant: 5).isActive = true
-
-        signalImage.centerYAnchor.constraint(equalTo: menuItemView.centerYAnchor).isActive = true
-        signalImage.heightAnchor.constraint(equalToConstant: 14).isActive = true
-        signalImage.rightAnchor.constraint(equalTo: menuItemView.rightAnchor, constant: -10).isActive = true
+    var heightConstraint: NSLayoutConstraint!
 
-        lockImage.centerYAnchor.constraint(equalTo: menuItemView.centerYAnchor).isActive = true
-        lockImage.rightAnchor.constraint(equalTo: signalImage.leftAnchor, constant: -10).isActive = true
-
-        ssidLabel.centerYAnchor.constraint(equalTo: menuItemView.centerYAnchor).isActive = true
-        ssidLabel.heightAnchor.constraint(equalTo: menuItemView.heightAnchor).isActive = true
-        ssidLabel.leftAnchor.constraint(equalTo: statusImage.rightAnchor, constant: -2).isActive = true
-        ssidLabel.rightAnchor.constraint(equalTo: lockImage.leftAnchor, constant: -10).isActive = true
+    func setupLayout() {
+        heightConstraint = heightAnchor.constraint(equalToConstant: 19)
+        heightConstraint.priority = NSLayoutConstraint.Priority(rawValue: 1000)
+        heightConstraint.isActive = true
+
+        statusImage.centerYAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
+        statusImage.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 6).isActive = true
+        statusImage.widthAnchor.constraint(equalToConstant: 12).isActive = true
+
+        ssidLabel.centerYAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
+        ssidLabel.leadingAnchor.constraint(equalTo: statusImage.trailingAnchor, constant: 3).isActive = true
+        ssidLabel.topAnchor.constraint(equalTo: self.topAnchor).isActive = true
+        ssidLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor).isActive = true
+
+        lockImage.centerYAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
+        lockImage.leadingAnchor.constraint(equalTo: ssidLabel.trailingAnchor, constant: 10).isActive = true
+        lockImage.widthAnchor.constraint(equalToConstant: 10).isActive = true
+
+        signalImage.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: 1).isActive = true
+        signalImage.leadingAnchor.constraint(equalTo: lockImage.trailingAnchor, constant: 12).isActive = true
+        signalImage.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -12).isActive = true
+        signalImage.widthAnchor.constraint(equalToConstant: 21).isActive = true
```

Comment:
Can you adjust the width here to 18? It will make the icon fit in the view better.